spec/models/product_importer_spec.rb

```diff
@@ -578,9 +578,12 @@
   describe "updating non-updatable fields on existing products" do
```

Comment:
Worth testing non-updatable fields on variants now too?

Reply:
This is actually testing the non-updatable fields on the variant, anyway I revisited this and realised we could remove the check for non-updatable product fields.